### ISSUE TITLE
Corpse death animations need a different `deadMins` value

### ISSUE THREAD

**Issue by necessarily-equal** [Unvanquished/Unvanquished#1260]
In practice, for the armorless human, this means that the dead body will teleport a bit higher once replaced.

**Comment by DolceTriade** [comment]
Yeah, we can have a config file with variable z offsets for the different death anims I guess... I wonder if we can re-export iqms with this set somehow if we parse the iqe file.

**Comment by illwieckz** [comment]
is it a duplicate of:

- #1058

?

**Comment by necessarily-equal** [comment]
#1058 was to be fixed by https://github.com/necessarily-equal/res-players_src.dpkdir/commit/7dc41cca04b8aef04d74c4d8b982f5dd5e613d70, but since https://github.com/Unvanquished/Unvanquished/commit/5de0b9289281793307e2bcc7aeb74779a866d678, something more elaborate is needed, to avoid (say) having one corpse over the ground and one under.
